service/src/test/java/uk/gov/hmcts/reform/fpl/utils/matchers/JsonMatcher.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.fpl.utils.matchers;
+
+import org.json.JSONObject;
+import org.mockito.ArgumentMatcher;
+import org.mockito.internal.matchers.ContainsExtraTypeInfo;
+import org.mockito.internal.matchers.text.ValuePrinter;
+import org.skyscreamer.jsonassert.JSONCompare;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.Map;
+
+import static org.mockito.internal.progress.ThreadSafeMockingProgress.mockingProgress;
+import static org.skyscreamer.jsonassert.JSONCompareMode.STRICT;
+
+public class JsonMatcher implements ArgumentMatcher<Object>, ContainsExtraTypeInfo, Serializable {
+
+    private final Map<String, Object> wanted;
+
+    public static Map<String, Object> eqJson(Map<String, Object> value) {
+        mockingProgress().getArgumentMatcherStorage().reportMatcher(new JsonMatcher(value));
```

Comment:
Do we need this? seems like Mockito internals as it's not properly documented.

Reply:
done